app/views/articles/_v2_form.html.erb

```diff
@@ -51,6 +49,12 @@
   </div>
 </div>
 
+<% cache("article-edit-scripts", expires_in: 8.hours) do %>
+  <script async>
+    <%= RunkitTag.script.html_safe %>
```

Comment:
could you replace this with `<%== RunkitTag.script %>`.

The double `==` means it's not going to be escaped, similar to what `.html_safe` does